src/components/TodosFilter/index.ts

```diff
@@ -0,0 +1 @@
+export * from './TodosFilter';
```

Comment:
The code looks good and doesn't have any issues related to the provided task description.